File: blink.c
// created by elliott roach
// created on feb 2026
// this blinks a light on and off


void setup() {
    // initializing pins
    pinMode(LED_BUILTIN, OUTPUT);
}


void loop() {
    // this blinks the light in infinent loop
    digitalWrite(LED_BUILTIN, HIGH);
    delay(1000);
    digitalWrite(LED_BUILTIN, LOW);
    delay(1000);
}

File: blink.py
"""
Made by Elliott Roach
Made on 2026 feb
this turns the light on and off
"""

import board
import digitalio
import time

# setup
led = digitalio.DigitalInOut(board.LED)
led.direction = digitalio.Direction.OUTPUT


while True:
    led.value = True
    time.sleep(1)
    led.value = False
    time.sleep(1)
    
